modules/katamari/src/test/ts/module/ephox/katamari/test/arb/ArbDataTypes.ts

```diff
@@ -7,10 +7,10 @@ import { Result } from 'ephox/katamari/api/Result';
 
 type Arbitrary<A> = fc.Arbitrary<A>;
 
-export const arbResultError = <A, E> (arbE: Arbitrary<E>): Arbitrary<Result<A, E>> =>
+export const arbResultError = <A = never, E = any> (arbE: Arbitrary<E>): Arbitrary<Result<A, E>> =>
```

Comment:
No need for an `any` here
```suggestion
export const arbResultError = <E, A = never> (arbE: Arbitrary<E>): Arbitrary<Result<A, E>> =>
```

Reply:
I'm not doing this, as it requires switching the order which is going to be confusing as the order for results is always `<Value, Error>`.